Survey/Survey/Models/SurveyViewModel.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import Combine
+
+class SurveyViewModel: ObservableObject {
+    @Published var inquiries: [Inquiry] = []
+    private var cancellables = Set<AnyCancellable>()
+    private let mockServer = MockServer()
+
+    func loadInquiries() {
+        mockServer.loadInquiries()
+            .receive(on: DispatchQueue.main)
+            .sink(receiveCompletion: { completion in
+                switch completion {
+                case .finished:
+                    break
+                case .failure(let error):
+                    print("Failed to load inquiries: \(error)")
```

Comment:
where is this printing?

Reply:
SurveyView, in case failed to load the question

Reply:
There's no console when you're running an application. That only works when you're running a console app, which this isn't. So in this case, you have two options:

1. log to file. There are 3rd party loggers for OSX we can install that will greatly simplify, just google around for a good library.
2. Show something to user. In general this is the preferred route if there's a recovery step they can take. (e.g. connect to the internet). If not, then yeah silent retry is better. But in this case, we don't want to bug the user, so we go straight to silent retry.

Bottom line is I think installing a logger library and using it would be the best approach.